.github/merge_rules.json

```diff
@@ -42,42 +52,25 @@
    },
    {
       "name": "OSS CI",
-      "patterns": [".github/**", ".circleci/**", ".jenkins/**", "scripts/**", "tools/**"],
-      "approved_by": ["ezyang", "pytorch/pytorch-dev-infra"],
-      "mandatory_checks_name": [
-         "Facebook CLA Check",
-         "Lint",
-         "pull"
-      ]
-   },
-   {
-      "name": "CI Pinned Hashes",
       "patterns": [
-         ".github/ci_commit_pins/vision.txt",
-         ".github/ci_commit_pins/torchdynamo.txt"
+         ".github/**",
+         ".circleci/**",
+         ".jenkins/**",
+         "scripts/**",
+         "tools/**",
+         ".github/ci_commit_pins/*"
       ],
-      "approved_by": ["pytorchbot", "ezyang", "pytorch/pytorch-dev-infra"],
+      "approved_by": ["alband", "dagitses", "pytorchbot", "pytorch/pytorch-dev-infra"],
       "mandatory_checks_name": [
          "Facebook CLA Check",
          "Lint",
          "pull"
       ]
    },
    {
-      "name": "XLA hash pin update",
-      "patterns": [".github/ci_commit_pins/xla.txt"],
-      "approved_by": ["pytorchbot", "ezyang", "pytorch/pytorch-dev-infra"],
-      "mandatory_checks_name": [
-         "Facebook CLA Check",
-         "Lint",
-         "pull / linux-bionic-py3_7-clang8-xla / build",
-         "pull / linux-bionic-py3_7-clang8-xla / test (xla, 1, 1, linux.2xlarge)"
-      ]
-   },
```

Comment:
Please keep separate rule for XLA hash pin update (but probably only for the bot, as XLA tests are sometimes disabled in pull and we don't want updates to happen during that time)

Reply:
done

Reply:
I created standalone rule only for the bot with the access to the folder with hash pin files